csharp/index.md

```diff
@@ -0,0 +1,4 @@
+# This is the **HOMEPAGE**.
+Refer to [Markdown](http://daringfireball.net/projects/markdown/) for how to write markdown files.
+## Quick Start Notes:
+1. Add images to the *images* folder if the file is referencing an image.
```

Comment:
I don't see this file being included anywhere in `docfx.json`

Reply:
move it to csharp/doc/index.md and it is used as the index of the generated doc